action.yml

```
@@ -32,6 +32,8 @@ runs:
   using: "composite"
   steps:
 
+    - uses: moneymeets/moneymeets-composite-actions/block-poetry-version-downgrade@feature/fix-sparse-checkout-error
```

```suggestion
    - uses: moneymeets/moneymeets-composite-actions/block-poetry-version-downgrade@master
```

Nice! The check is working correctly when the version is downgraded, https://github.com/moneymeets/moneymeets-pulumi/actions/runs/7929170398/job/21648873954. I'm approving, just making the suggestion so that we don't forget to use master here, once the fix in composite-actions is merged 👍